eventPage.js

```diff
@@ -10,7 +10,7 @@ function startApplication() {
         id: 'main-window',
         frame: 'chrome',
         innerBounds: {
-            minWidth: 960,
+            minWidth: 1020,
```

Comment:
This was changed to give space for the dropdown without pushing the gyro/bar/etc status boxes down while the documentation link is also visible.  I don't have a chromebook to try it on, but since the width on the 13" model is 1366, I think it should be ok.